tests/test_bert_vectorizer.py

```diff
@@ -12,43 +12,50 @@
 ]
 
 
-def test_embed_one_sentence():
+@pytest.fixture
```

Comment:
I think it will make tests even quicker to invoke once by using scope module https://docs.pytest.org/en/latest/how-to/fixtures.html#fixture-scopes